docs/platform-support.rst

```diff
@@ -15,8 +15,95 @@
   limitations under the License.
 
 .. headings # #, * *, =, -, ^, "
-
+.. include:: ./common/vars.rst
 
 ****************
 Platform Support
-****************
\ No newline at end of file
+****************
+
+*******************
+System Requirements
+*******************
+
+* NVIDIA RDMA-capable network adapters:
+   * NVIDIA ConnectX NICs
+        * ConnectX-5 or newer
+   * NVIDIA BlueField Network Platforms
+        * BlueField-2 DPU (NIC mode)
+        * BlueField-3 DPU (NIC mode)
+        * BlueField-3 SuperNIC (NIC mode)
+* NVIDIA GPU Operator Version 23.9.x or newer (required for the workloads using NVIDIA GPUs and GPUDirect RDMA technology)
+* Operating Systems:
+    * Ubuntu: v22.04, v20.04
+    * OpenShift Container Platform (OCP): v 4.14, v4.13, v4.12
+    * RHEL: v9.2, v9.1, v8.8, v8.6
+* Container runtime: containerd, CRI-O
+
+***********************
+Tested Network Adapters
+***********************
+The following network adapters have been tested with the Network Operator:
+
+* ConnectX-6 Dx
+* ConnectX-7
+* BlueField-2 NIC Mode
+* BlueField-3 NIC Mode
+
```

Comment:
Also missing 'Supported ARM Based Platforms'
Make sure to look at 24.1 page.
https://docs.nvidia.com/networking/display/kubernetes2410/platform+support